src/Spout/Writer/Common/Helper/AppendHelper.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace Box\Spout\Writer\Common\Helper;
+
+class AppendHelper {
+
+    /**
+     * Instead of seeking and re-writing from position, a better hack might be to write dummy empty data
+     * Enough to take care of any length, then carefully overwrite
+     * 
+     */
+
+    /**
+     * This function will truncate from specified position
+     * Write data to be inserted and re-append the truncated data
+     *  
+     * @param $fp Pointer to file only
+     * @param $pos Position to insert
+     * @param $content Content to insert
+     */
+    public static function insertToFile($fp, $pos, $content)
```

Comment:
This can be very costly to rewrite the contents, especially with large spreadsheet

Reply:
True, I will try both approaches and do a benchmark today. It didn't seem to matter for small files.

Reply:
@adrilo So I ran some tests and this is not actually a bad solution at all... (performance differences range from 3% to 7% and seems to get better for larger amount of rows).. (Only tested 100,000, 500,000 and 1 million rows though)

The difference are quite negligible most likely because the file is still in memory the whole time and the only costly operation here is really calling `stream_get_contents`.

![image](https://user-images.githubusercontent.com/3619843/153718720-5b2c7ecc-8a32-4f2e-9836-4e26efec6565.png)
![image](https://user-images.githubusercontent.com/3619843/153718943-39ac990b-e560-41ea-b709-4681894e1261.png)